store/tikv/client_batch.go

```diff
@@ -249,23 +252,33 @@ func (c *batchCommandsClient) batchRecvLoop(cfg config.TiKVClient) {
 	for {
 		resp, err := c.recv()
 		if err != nil {
+			b := NewBackoffer(context.Background(), math.MaxInt32)
 			now := time.Now()
-			for { // try to re-create the streaming in the loop.
+			skipLog := false
+			for count := 0; ; count++ { // try to re-create the streaming in the loop.
 				if c.isStopped() {
 					return
 				}
-				logutil.BgLogger().Error(
-					"batchRecvLoop error when receive",
-					zap.String("target", c.target),
-					zap.Error(err),
-				)
 
-				if c.reCreateStreamingClient(err) {
-					break
+				if count <= 10 {
+					logutil.BgLogger().Error(
+						"batchRecvLoop error when receive",
+						zap.String("target", c.target),
+						zap.Error(err),
+					)
```

Comment:
The log seems to be duplicate to the log in `reCreateStreamingClient`.
Perhaps it should appear once after `c.recv` fires error.
```go
for {
    resp, err := c.recv()
    if err != nil {
        // log here because `c.recv()` fires error.
        for {
            // recreate streaming client
        }
    }
}
```

Reply:
The error is not the same, one is `c.recv()` error, and the other is `recreate` error

Reply:
Get it!